allennlp/commands/fine_tune.py

```diff
@@ -57,6 +57,12 @@ def add_subparser(self, name: str, parser: argparse._SubParsersAction) -> argpar
                                help='a JSON structure used to override the training configuration '
                                '(only affects the config_file, _not_ the model_archive)')
 
+        subparser.add_argument('--extend-vocab',
+                               action='store_true',
+                               default=False,
```

Comment:
nit: I don't think this default is necessary when `action='store_true'`

Reply:
explicit is better than implicit